### PR TITLE
YugabyteDB Managed Integration

### PR DESCRIPTION
**Pull request by daniel-yb**
### What does this PR do?

Adding an integration tile for `YugabyteDB Managed` clusters including a pre-built dashboard.

### Motivation

YugabyteDB Managed has introduced an offering to export metrics from clusters -> datadog.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: yugabytedb_managed/manifest.json
Comment (daniel-yb) [inline yugabytedb_managed/manifest.json:1]: Not sure why this is complaining. I've added two svg files to `assets/logos` as specified in https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files. I've also ran the validate cmd locally without any errors returned:
```
$ ddev validate all yugabytedb_managed
Running validations for extras repo ...
---
Skipping agent-reqs
---
Executing validation codeowners
All integrations have valid codeowners.

---
Executing validation config
Validating default configuration files for 1 checks...
yugabytedb_managed:
Did not find spec file /Users/daniel/dd/integrations-extras/yugabytedb_managed/ for check yugabytedb_managed

---
Executing validation dashboards
Validating Dashboard definition files for 1 checks...
1 valid files

---
Skipping dep
---
Skipping eula
---
Executing validation jmx-metrics
Validating JMX metrics files for 0 checks ...
0 total JMX integrations
0 valid metrics files

---
Executing validation imports
Validating imports for 1 checks to avoid deprecated modules ...
Validation passed!

---
Executing validation manifest
Validating manifest.json files for 1 checks ...

---
Executing validation models
Validating data models for 1 checks ...

---
Executing validation package
Validating files for 0 checks ...

---
Executing validation readmes
All 1 READMEs are valid!

---
Executing validation saved-views
Validating saved views for 1 checks ...
All saved views are valid!

---
Executing validation service-checks
Validating service_checks.json files for 1 checks ...
1 valid files
```

**Comment by aliciascott** [comment]
Thanks @daniel-yb for the add, I have created an Editorial Review card for our Docs team to conduct a more in-depth review.

File: yugabytedb_managed/README.md
Comment (emarsha94) [inline yugabytedb_managed/README.md:5]: The overview should include:

1. What does your product do in 1-2 sentences? 
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data?

File: yugabytedb_managed/README.md
Comment (emarsha94) [inline yugabytedb_managed/README.md:33]: You can include in the overview that this integration comes with an out-of-the-box dashboard to provide visibility into the YugabyteDB Managed clusters

File: yugabytedb_managed/manifest.json
Comment (emarsha94) [inline yugabytedb_managed/manifest.json:13]: Please include images for your integration tile. We highly recommend images of the graphs in the dashboard!

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Thanks for submitting your PR! We are excited about the integration. I left some comments on the PR. Please let me know if you have any questions

File: yugabytedb_managed/README.md
Comment (daniel-yb) [reply yugabytedb_managed/README.md]: updated.

File: yugabytedb_managed/README.md
Comment (daniel-yb) [reply yugabytedb_managed/README.md]: updated.

File: yugabytedb_managed/manifest.json
Comment (daniel-yb) [reply yugabytedb_managed/manifest.json:13]: Will work on getting this piece done first thing next week.

**Comment by daniel-yb** [comment]
> Thanks for submitting your PR! We are excited about the integration. I left some comments on the PR. Please let me know if you have any questions

Thanks for the feedback so far @emarsha94 ! Looking forward to working with you and the Datadog team to get this pushed through.

File: yugabytedb_managed/manifest.json
Comment (daniel-yb) [reply yugabytedb_managed/manifest.json]: Done.

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:38]:
```suggestion
In the Datadog dashboard list search for and select the `YugabyteDB Managed Overview` dashboard to view your telemetry.
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:43]:
```suggestion
1. In YugabyteDB Managed, navigate to the **Integrations > Metrics** tab.
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:46]:
```suggestion
4. Wait for the **Export Status** to show `-`. <br> **Note**: Your cluster cannot associate a configuration when paused or when another operation in progress.
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:47]:
```suggestion
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:53]:
```suggestion
See [metadata.csv][6] for a list of metrics provided by this integration.
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:65]:
```suggestion
Need help? Contact [YugabyteDB support][7].
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:67]:
```suggestion
[1]: https://yugabyte.com/
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:68]:
```suggestion
[2]: https://www.yugabyte.com/managed/
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:69]:
```suggestion
[3]: https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-monitor/metrics-export/#datadog
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:70]:
```suggestion
[4]: https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:71]:
```suggestion
[5]: https://docs.datadoghq.com/getting_started/site/
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:72]:
```suggestion
[6]: https://github.com/DataDog/integrations-extras/blob/master/yugabytedb_managed/metadata.csv
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:73]:
```suggestion
[7]: https://support.yugabyte.com/
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:74]:
```suggestion
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:75]:
```suggestion
```

**Review by estherk15 — COMMENTED** [review]
Thanks for your work on this! Let me know if you have any questions on my review of the docs!

**Comment by daniel-yb** [comment]
> Thanks for your work on this! Let me know if you have any questions on my review of the docs!

Thanks for the review @estherk15 ! I've updated the PR with your suggestions.

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:14]: I would leave this as regular text, this doesn't limit customer ability to use this integration. Add this link to the bottom of the page https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-monitor/metrics-export/#datadog
```suggestion
For more information on setup, see the [YugabyteDB documentation][x]
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:24]:
```suggestion
4. Fill in the **API key** and **Site** fields with the corresponding values. For more information, see the [Datadog API and Application Keys][3] and [Datadog Site URL][4] documentation.
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:25]:
```suggestion
```

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:26]:
```suggestion
```

**Review by estherk15 — COMMENTED** [review]
Thanks for making those changes. A few more nits. Here's a [Preview](https://docs-staging.datadoghq.com/esther/yugabytedb_managed-preview/integrations/yugabytedb_managed/)

File: yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json
Comment (emarsha94) [inline yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json:9]: The logo needs to be either png, gif, or jpg. Please update the image to one of the listed formats

File: yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json
Comment (emarsha94) [inline yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json:338]:
```suggestion
        "content": "#### This dashboard enables monitoring of YugabyteDB Managed clusters on telemetry such as:\n- Infrastructure related metrics (CPU usage, memory usage, etc)\n- YSQL related metrics (ops/sec, latencies, etc)\n- YCQL related metrics (ops/sec, latencies, etc)\n- Tablet server (tserver) related metrics\n- Master server (master) related metrics",
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Thanks for submitting the integration! Requested some changes. Please let me know if you have any questions!

File: yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json
Comment (daniel-yb) [reply yugabytedb_managed/assets/dashboards/yugabytedb_managed_overview.json]: Done

File: yugabytedb_managed/README.md
Comment (daniel-yb) [reply yugabytedb_managed/README.md]: Done.

**Comment by daniel-yb** [comment]
I've gone through and updated the PR with all of your comments @emarsha94 and @estherk15 . Appreciate both of your time and help on this!

File: yugabytedb_managed/README.md
Comment (estherk15) [inline yugabytedb_managed/README.md:15]: This one's my fault, forgot the punctuation.
```suggestion
For more information on setup, see the [YugabyteDB documentation][3].
```

**Review by estherk15 — APPROVED** [review]

File: .github/CODEOWNERS
Comment (emarsha94) [inline .github/CODEOWNERS:196]: please add @DataDog/ecosystems-review

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Minor changes left! Once that is done, we should be good to approve from our end

**Review by alopezz — APPROVED** [review]

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:13]: Just as a heads up, this will not render on the integration tile in-app, it will only render in the documentation page. If you'd like it to render in-app, you'll need to remove the HTML bits and just add it as plain markdown.

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:15]: I'd recommend moving this to the bottom of the installation section

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:33]:
```suggestion
4. Wait for the **Export Status** to show `Active`.*

*Note**: Your cluster cannot associate a configuration when paused or when another operation in progress.
```

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:41]:
```suggestion
4. Wait for the **Export Status** to show `-`. 

**Note**: Your cluster cannot associate a configuration when paused or when another operation in progress.
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
This is looking great, just a few more minor comments from my end. Thanks!

File: yugabytedb_managed/README.md
Comment (estherk15) [reply yugabytedb_managed/README.md:13]:
```suggestion
**Note**: This feature is not available for [Sandbox Clusters][X].
```
[X]: https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-free/

@daniel-yb that's on me, I was only thinking of the documentation!

Comment (daniel-yb) [reply yugabytedb_managed/README.md:13]: All good! 😄

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:33]:
```suggestion
**Note**: Your cluster cannot associate a configuration when paused or when another operation is in progress.
```

File: yugabytedb_managed/README.md
Comment (anawishnoff) [inline yugabytedb_managed/README.md:45]:
```suggestion
**Note**: Your cluster cannot associate a configuration when paused or when another operation is in progress.
```

**Review by anawishnoff — APPROVED** [review]
Looks good to me other than two minor fixes!

**Comment by anawishnoff** [comment]
Hi @daniel-yb, @emarsha94 is out of office so I took over reviewing for her. We just need one last engineering review which will take place on Monday and then we can wrap this up and do a live demo. Thanks for your patience here!

**Comment by emarsha94** [comment]
Hi @daniel-yb we have reached out to engineering for review. Thanks!

**Review by JoshPatel13 — APPROVED** [review]

**Comment by daniel-yb** [comment]
@emarsha94 @anawishnoff now that the PR has approval, what are the next steps? Thanks!

**Comment by emarsha94** [comment]
@daniel-yb Would you like to test this in your sandbox first or publish live?

**Comment by daniel-yb** [comment]
@emarsha94 please feel free to go live with it.

File: yugabytedb_managed/metadata.csv
Comment (emarsha94) [inline yugabytedb_managed/metadata.csv:1]: @daniel-yb Apologies - last change and then we are good to go. Can you please include descriptions for the metrics? This is a requirement for integrations sending in metrics. Once that is done, we can merge as soon as Monday. Thanks!

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: yugabytedb_managed/metadata.csv
Comment (daniel-yb) [reply yugabytedb_managed/metadata.csv:1]: Sure. Your [docs](https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file) explicitly say the description is optional, that should probably be updated to avoid future confusion with others.

**Comment by daniel-yb** [comment]
Hi @JoshPatel13 thank you for merging the PR. While I can see the docs live at https://docs.datadoghq.com/integrations/yugabytedb_managed/, when can I expect to see the integration show up when searching through https://app.datadoghq.com/integrations? Thanks!

**Pull request merged**
